tools/github/contributions.sh

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env bash
+
+parse_args() {
+    if [ -z "$2" ]; then
+        echo "value for parameter '$1' cannot be empty"
+        exit 1
+    fi
+    case "$1" in
+        -t)
```

Comment:
Each of these could be more verbose, like `--prtype|-t)` and `--newbranch|-n` so they have a long and shorthand format.